packages/neutrino-preset-node/index.js

```diff
@@ -25,7 +25,7 @@ module.exports = (neutrino) => {
     Object.assign(neutrino.options, {
       compile: {
         targets: {
-          node: 6.9
+          node: 6.10
```

Comment:
Since this is a number, it will evaluate to `6.1`. This probably needs to be a string now `'6.10'`.